CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Revision history for lumberjack
 
+## 1.0.3.0 -- 2023-07-28
+
+  * Update for GHC 9.6 (`base-4.18.*`).
+  * Remove `lumberjack`'s dependency on `mtl`, which went unused.
```

Comment:
I don't see a change associated with this item?

Reply:
This was done in 298f84243df631dc58791c7e41af846913bf10c6, but I forgot to mention it in the `CHANGELOG` at the time due to an oversight.